community/udc/src/main/filtered-resources/org/neo4j/ext/udc/udc.properties

```diff
@@ -1,3 +0,0 @@
-unsupported.dbms.udc.source=${neo4j-source-distribution}
```

Comment:
Should we not also add this injection to neo4j.conf? Otherwise we won't be collecting distribution information at all.

Reply:
If we remove this file, these two https://github.com/neo4j/neo4j/blob/3.3/community/udc/pom.xml#L68 and https://github.com/neo4j/neo4j/blob/3.3/community/udc/pom.xml#L89 are useless, no?

Reply:
@chrisvest we have an injection of this property in neo4j.conf when we do packaging:
dbms.jvm.additional=-Dunsupported.dbms.udc.source=#{unsupported.dbms.udc.source}
and "unsupported.dbms.udc.source" is replaced with tarball or zip when we do standard packaging for example

Reply:
@davidegrohmann yes you are right those 2 profiles are useless and always were and they already removed in 3.4 development branch